### PR TITLE
Update questions with new wording

### PR DESCRIPTION
**Pull request by davidrapson**
Updates BCF questions with new wording, notably merging the idea, impact, activities into a single screen with smaller questions.

### CODE REVIEW

File: controllers/apply/building-connections/fields.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const { check } = require('express-validator/check');
 
 const LENGTH_HINTS = {
-    ONE_LINER: {
+    HEMINGWAY: {
```
Comment (davidrapson) [inline controllers/apply/building-connections/fields.js]: We never use this length but I couldn't resist.

Comment (mattandrews) [reply controllers/apply/building-connections/fields.js]: 💯

File: controllers/apply/building-connections/fields.js [hunk 2]
```diff
@@ -45,49 +46,69 @@ const currentWork = [
...
                 type: 'textarea',
-                lengthHint: LENGTH_HINTS.FEW_LINES,
+                lengthHint: LENGTH_HINTS.FEW_PARAS,
                 isRequired: true,
-                label: 'What will be achieved in the first quarter Jan—March 2019?'
-            },
+                label: 'How will the project help to build social connections and relationships between individuals and communities?',
+                helpText: `<ul>
+                    <li>Describe how the project will increase social connections, sense of community and/or belonging</li>
+                    <li>Describe how are people and communities involved in the project</li>
```
Comment (mattandrews) [inline controllers/apply/building-connections/fields.js]: I'd phrase this like:
"Describe how people and communities are involved in the project"

Comment (davidrapson) [reply controllers/apply/building-connections/fields.js]: Good shout.

**Pull request merged**